README.md

```diff
@@ -1,3 +1,13 @@
 # GlobalGiving Depth
 
 Gain a depth in insight into NGOs discovered through an automated process.
+
```

Comment:
I very much appreciate the readme! However, this readme doesn't pertain to the entire repo, so it would be good to move it into an appropriate subdirectory in src/ along with the other files